vscode-dotnet-runtime-library/src/test/unit/ExistingPathResolver.test.ts

```diff
@@ -3,6 +3,7 @@
 *  The .NET Foundation licenses this file to you under the MIT license.
 *--------------------------------------------------------------------------------------------*/
 import * as chai from 'chai';
+import * as os from 'os';

```

Comment:
Can test code be moved next to other test code?

Reply:
I'm not sure what you mean - maybe you mean if this import can be added to when it needs to be used? The typical style is to put them at the top. But probably the answer is yes